HowToPlay.html

```diff
@@ -63,16 +63,40 @@
 			font-size: 20px;
 			text-indent: 20px;
 			font-weight: bold;
+		}
```

Comment:
Are the changes on this file necessary?

Reply:
I did not change this part in this push, but these lines are necessary.
There exactly some other changes in this HowToPlay.html just for test but not necessary, so please use the the prior version of this file, sorry for the mistakes.